packages/variant-theme/package.json

```diff
@@ -2,13 +2,17 @@
   "name": "@fluentui-contrib/variant-theme",
   "version": "0.0.1",
   "dependencies": {
-    "@swc/helpers": "~0.5.2"
+    "@swc/helpers": "~0.5.2",
+    "@fluentui/react-components": ">=9.46.3 <10.0.0",
+    "@fluentui/react-migration-v8-v9": "^9.6.12",
+    "@fluentui/scheme-utilities": "^8.3.46",
+    "@griffel/react": "^1.5.14",
+    "lodash" :"^4.17.21"
```

Comment:
unused - pls remove / note nowadays there is very little need to use lodash as most functionality is build in into ECMA already. lets not pollute users bundle-sizes with junk

Reply:
removed lodash